web/src/utils/getSnapPoints.ts

```diff
@@ -1,4 +1,9 @@
-const min = 0.1
+const min = 0.05
 const mid = 0.35
 const max = 0.65
-export const getSnapPoints = (maxHeight: number): number[] => [maxHeight * min, maxHeight * mid, maxHeight * max]
+export const getSnapPoints = (maxHeight: number): number[] => [
```

Comment:
Maybe you can eliminate some non null checks with this:
```suggestion
export const getSnapPoints = (maxHeight: number): [number, number, number, number] => [
```